infra-templates/ipsec/15/docker-compose.yml.tpl

```diff
@@ -41,6 +41,11 @@ services:
       net.ipv4.conf.default.send_redirects: '0'
       net.ipv4.conf.eth0.send_redirects: '0'
       net.ipv4.xfrm4_gc_thresh: '2147483647'
+  hc:
+    image: leodotcloud/self-health-status:dev
```

Comment:
what is this image going to be?

Reply:
I will package the binary in `rancher/net`